Makefile

```diff
@@ -68,9 +69,9 @@ secvarctl: $(OBJ)
 	$(CC) $(CFLAGS) $(_CFLAGS) -c  $< -o $@
 
 clean:
-	rm -f $(OBJ) secvarctl 
-	rm -f $(OBJ:.o=.d)
-	rm -f ./*/*.cov.* secvarctl-cov ./*.cov.* ./backends/*/*.cov.* ./external/*/*.cov.* ./html*
+	find . -name "*.[od]" -delete
+	find . -name "*.cov.*" -delete
```

Comment:
`find` is included in distros right? I refuse to put it in package requirements hehehe

Reply:
Regardless, this is the one-stop-cleanup-shop command I was looking for. Thanks!